docs/sources/setup-grafana/installation/helm/index.md

```diff
@@ -0,0 +1,374 @@
+
+When you deploy Grafana Helm charts, use a separate namespace instead of relying on the default namespace. The default namespace might already have other applications running, which can lead to conflicts and other potential issues.
+
+When you create a new namespace in Kubernetes, you can better organize, allocate, and manage cluster resources. For more information about Namespaces, refer to [Namespaces](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/).
+
+1. To create a namespace, run the following command:
+
+   ```bash
+   kubectl create namespace monitoring
+   ```
+
+   You will see an output similar to this, which means that the namespace has been successfully created:
+
+   ```bash
+   namespace/monitoring created
+   ```
+
+2. Search for the official `grafana/grafana` repository using the command:
+
+   `helm search repo <repo-name/package-name>`
+
+   For example, the following command provides a list of the Grafana Helm Charts from which you will install the latest version of the Grafana chart.
+
+   ```bash
+   helm search repo grafana/grafana
+   ```
+
+3. Run the following command to deploy the Grafana Helm Chart inside your created namespace.
+
+   ```bash
+   helm install my-grafana grafana/grafana --namespace monitoring
+   ```
+
+   Where:
+   - `helm install`: installs the chart by deploying it on the Kubernetes cluster
+   - `my-grafana`: the logical chart name that we had given
+   - `grafana/grafana`: the repository and package name to install
+   - `--namespace`: the Kubernetes namespace (i.e. `monitoring`) where you want to deploy the chart
+
+4. To verify the deployment status, run the following command and verify that `deployed` appears in the **STATUS** column:
+
+   ```bash
+   helm list -n monitoring
+   ```
+
+   You should see an output similar to the following:
+
+   ```bash
+   NAME            NAMESPACE       REVISION        UPDATED                                 STATUS          CHART          APP VERSION
+   my-grafana      monitoring      1               2024-01-13 23:06:42.737989554 +0000 UTC deployed        grafana-6.59.0 10.1.0   
+   ```
+
+5. To check the overall status of all the objects in the namespace, run the following command:
+
+      ```bash
+      kubectl get all -n monitoring
+      ```
+
+   If you encounter errors or warnings in the **STATUS** column, check the logs and refer to Troubleshooting setction of this documentation.
+
+### Accessing Grafana
+
+This section describes the steps you must complete to access Grafana via web browser.
+
+First, run the following `helm status` command:
+
+```bash
+helm status my-grafana -n monitoring
+```
+
+This command will give you the complete status of the release information such as last deployment time, namespace (where it is deployed), release status, additional notes provided by the chart etc.
+
+Within the additonal release notes, it provides the following 2 key information i.e.
```

Comment:
@usmangt I'm having trouble following this. When I run the helm status command you say it provides the following 2 key information. Does that command literally have the words "How do decode the login password for the admin account"? Or is there something in that output that the user needs to make note of so that they can use it in the next command, which is the command on like 159?